{"name": "I try to log in with unregistered account -- @1.2 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=122.0.6261.129)\nStacktrace:\n0   chromedriver                        0x0000000104e60524 chromedriver + 3966244\n1   chromedriver                        0x0000000104e58ab0 chromedriver + 3934896\n2   chromedriver                        0x0000000104adbda0 chromedriver + 277920\n3   chromedriver                        0x0000000104ab6fb0 chromedriver + 126896\n4   chromedriver                        0x0000000104b43414 chromedriver + 701460\n5   chromedriver                        0x0000000104b56304 chromedriver + 779012\n6   chromedriver                        0x0000000104b12fb0 chromedriver + 503728\n7   chromedriver                        0x0000000104b13a28 chromedriver + 506408\n8   chromedriver                        0x0000000104e2570c chromedriver + 3725068\n9   chromedriver                        0x0000000104e29c00 chromedriver + 3742720\n10  chromedriver                        0x0000000104e0e1f4 chromedriver + 3629556\n11  chromedriver                        0x0000000104e2a6fc chromedriver + 3745532\n12  chromedriver                        0x0000000104e0156c chromedriver + 3577196\n13  chromedriver                        0x0000000104e48f5c chromedriver + 3870556\n14  chromedriver                        0x0000000104e49100 chromedriver + 3870976\n15  chromedriver                        0x0000000104e58720 chromedriver + 3933984\n16  libsystem_pthread.dylib             0x000000018ab46f94 _pthread_start + 136\n17  libsystem_pthread.dylib             0x000000018ab41d34 thread_start + 8\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/bookstore_login_steps.py\", line 6, in step_impl\n    context.bookstore_login_page.open()\n  File \"/Users/olaru/Projects/ProiectFinal/pages/bookstore_login_page.py\", line 15, in open\n    self.driver.get(self.URL_LOGIN)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 357, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I am on the Login page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=122.0.6261.129)\nStacktrace:\n0   chromedriver                        0x0000000104e60524 chromedriver + 3966244\n1   chromedriver                        0x0000000104e58ab0 chromedriver + 3934896\n2   chromedriver                        0x0000000104adbda0 chromedriver + 277920\n3   chromedriver                        0x0000000104ab6fb0 chromedriver + 126896\n4   chromedriver                        0x0000000104b43414 chromedriver + 701460\n5   chromedriver                        0x0000000104b56304 chromedriver + 779012\n6   chromedriver                        0x0000000104b12fb0 chromedriver + 503728\n7   chromedriver                        0x0000000104b13a28 chromedriver + 506408\n8   chromedriver                        0x0000000104e2570c chromedriver + 3725068\n9   chromedriver                        0x0000000104e29c00 chromedriver + 3742720\n10  chromedriver                        0x0000000104e0e1f4 chromedriver + 3629556\n11  chromedriver                        0x0000000104e2a6fc chromedriver + 3745532\n12  chromedriver                        0x0000000104e0156c chromedriver + 3577196\n13  chromedriver                        0x0000000104e48f5c chromedriver + 3870556\n14  chromedriver                        0x0000000104e49100 chromedriver + 3870976\n15  chromedriver                        0x0000000104e58720 chromedriver + 3933984\n16  libsystem_pthread.dylib             0x000000018ab46f94 _pthread_start + 136\n17  libsystem_pthread.dylib             0x000000018ab41d34 thread_start + 8\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/bookstore_login_steps.py\", line 6, in step_impl\n    context.bookstore_login_page.open()\n  File \"/Users/olaru/Projects/ProiectFinal/pages/bookstore_login_page.py\", line 15, in open\n    self.driver.get(self.URL_LOGIN)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 357, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1710952535558, "stop": 1710952535559}, {"name": "When I type \"testing\" in the username input", "status": "skipped", "start": 1710952535560, "stop": 1710952535560}, {"name": "And I type \"testpass9\" in the password input", "status": "skipped", "start": 1710952535560, "stop": 1710952535560}, {"name": "And I click the login button", "status": "skipped", "start": 1710952535560, "stop": 1710952535560}, {"name": "Then Login error message appears", "status": "skipped", "start": 1710952535560, "stop": 1710952535560}], "parameters": [{"name": "username", "value": "testing"}, {"name": "password", "value": "testpass9"}], "start": 1710952535558, "stop": 1710952535560, "uuid": "eee18353-b836-4e6b-a2e3-69947f287533", "historyId": "085810e48851b8914b3004468ba609ac", "fullName": "Login Page: I try to log in with unregistered account", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "LoginPage"}, {"name": "tag", "value": "negative"}, {"name": "tag", "value": "smoke"}, {"name": "feature", "value": "Login Page"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}